pokemongo_bot/cell_workers/evolve_pokemon.py

```diff
@@ -66,12 +66,15 @@ def work(self):
                 pokemon_to_be_evolved = pokemon_to_be_evolved + min(candy.quantity / (pokemon.evolution_cost - 1), filtered_dict[pokemon.pokemon_id])
 
         if pokemon_to_be_evolved >= self.min_pokemon_to_be_evolved:
+            self.logger.info("Checking minimum pokemon to evolve: {}/{}. Hooray!".format(pokemon_to_be_evolved, self.min_pokemon_to_be_evolved))
             if self.use_lucky_egg:
                 self._use_lucky_egg()
             cache = {}
             for pokemon in filtered_list:
                 if pokemon.can_evolve_now():
                     self._execute_pokemon_evolve(pokemon, cache)
+        else:
+            self.logger.info("Checking minimum pokemon to evolve: {}/{}. Not enough, gotta catch`em all (over and over again)".format(pokemon_to_be_evolved, self.min_pokemon_to_be_evolved))
```

Comment:
Again, do we need humorous status messages? Not a fan.

Suggest "Not enough pokemon for evolve" would do.